deploy/infraagent-configmap.yaml

```diff
@@ -6,3 +6,4 @@ metadata:
 data:
   infraagent.yaml: |
     interfaceType: tap
+    insecure: true
```

Comment:
Instead of insecure and secure  & TLS and mTLS, my preference would be 1) variable - "mode" 
mode =insecure, mode =TLS, mode=mTLS